recipe_scrapers/joshuaweissman.py

```diff
@@ -0,0 +1,70 @@
+# mypy: allow-untyped-defs
+
+# from pprint import pprint
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class JoshuaWeissman(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "joshuaweissman.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        raise NotImplementedError()
+
+    def yields(self):
+        raise NotImplementedError()
```

Comment:
This seems OK to me - there is servings-related info on the page, but it's tricky to extract, so we can leave this to be implemented later.

I don't see any time-related info for the test recipe -- so we could `return None` there -- but that likely only makes sense if we're fairly sure that there aren't any pages that _do_ have time info available.

Reply:
It seems like most recipes have neither, but some have serving size and some have time. As they are all formatted about the same as everything else in the recipes, it would probably only work as trying to find exact text and grab that tag. That would probably only work sometimes but there isn't much overall for it

Reply:
Added this and a naive total_time